main.go

```diff
@@ -3,42 +3,104 @@ package main
 import (
 	"context"
 	"encoding/json"
+	"fmt"
+	"log/slog"
 	"os"
 	"strings"
 
 	"github.com/jomei/notionapi"
+	"github.com/urfave/cli/v2"
 )
 
-func main() {
-	notionKey := os.Getenv("NOTION_SECRET")
-	repoPath := os.Args[1]
+var app = cli.App{
+	Name:  "notionreposync",
+	Usage: "notionreposync --page-id <notion-page-id> --api-key <notion-api-key> <repo-path>",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:     "page-id",
+			Required: true,
+			EnvVars:  []string{"NOTION_PAGE_ID"},
+			Action: func(ctx *cli.Context, v string) error {
+				if v == "" {
+					return fmt.Errorf("page-id (NOTION_PAGE_ID) cannot be empty")
```

Comment:
```suggestion
					return fmt.Errorf("page-id ($NOTION_PAGE_ID) cannot be empty")
```

Should we add a `$` so it's evident that it's an environment variable.

Reply:
agreed